README.md

```diff
@@ -71,8 +70,8 @@ $ npx jasmine-browser-runner serve
 1. Update the version in `manifest.json`
 2. Run `npm run build`
 3. Create a Pull Request with the new package committed
-4. Upload newly created package in `/build` to the [Chrome web store](https://chrome.google.com/webstore/devconsole/06b3913d-07a7-479e-94aa-05bb5b3cd44d/dclfaikcemljbaoagjnedmlppnbiljen/edit/package).
-5. Upload newly created package in `/build` to [Firefox Add-ons](https://addons.mozilla.org/en-US/developers/addon/govuk-browser-extension-ff/versions/submit/). Account details in the [2nd line password store](https://github.com/alphagov/govuk-secrets/tree/master/pass/2ndline/firefox/add-ons).
+4. Upload newly created package created for chrome at`/build/govuk-browser-extension-chrome-{version}.zip` to the [Chrome web store](https://chrome.google.com/webstore/devconsole/06b3913d-07a7-479e-94aa-05bb5b3cd44d/dclfaikcemljbaoagjnedmlppnbiljen/edit/package).
```

Comment:
Just under "Releasing the extension", can you add a paragraph explaining why we have to deploy two different versions of the extension?